nodejs/eks/utils.ts

```diff
@@ -21,3 +22,14 @@ import * as pulumi from "@pulumi/pulumi";
  * https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html.
  */
 export type InputTags = pulumi.Input<{ [key: string]: pulumi.Input<string> }>;
+
+/**
+ * UserStorageClasses represents a map of an Input type to a
+ * k8s StorageClass to capture all of the *user-created* storage classes. This
+ * map does not include the default `gp2` automatically added by EKS.
+ *
+ * Note: As of Kubernetes v1.11+ on EKS, a default `gp2` storage class will
+ * always be created automatically for the cluster by the EKS service. See
+ * https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html
+ */
+export type UserStorageClasses = pulumi.Input<{ [key: string]: k8s.storage.v1.StorageClass }>;
```

Comment:
Since this only appears in output positions, I assume it would be more useful to make it an `Output`?

Reply:
Done